orttraining/orttraining/test/python/orttraining_test_checkpoint.py

```diff
@@ -131,37 +119,11 @@
 _single_run(load_checkpoint_file, 'test_load_from_distributed_zero_megatron_mixed_precision_into_single_node_mixed_precision', distributed_zero_megatron_mixed_precision_lamb_path)
 _single_run(load_checkpoint_file, 'test_load_from_distributed_zero_megatron_full_precision_into_single_node_mixed_precision', distributed_zero_megatron_full_precision_lamb_path)
 
-# going to data parallel trainer
-_distributed_run(load_checkpoint_file, 'test_load_from_single_node_full_precision_into_data_parallelism_full_precision', single_node_full_precision_path)
```

Comment:
I feel we should not remove loading into data parallelism since we are testing whether all the nodes have loaded the checkpoint correctly.
I think loading into data parallelism is not the same as loading into a single node tainer? What are your thoughts on this?